packages/common/src/database/DbConnection.ts

```diff
@@ -32,8 +33,27 @@ function addRepositorySchema(query: string, repositoryData: RepositoryData) {
 export class DbConnection {
     postgresConnection: pgPromise.IDatabase<object, pg.IClient>
     dbConnection: pgPromise.IDatabase<object, pg.IClient>
-    pgp: pgPromise.IMain<object, pg.IClient>
+    private _pgp: pgPromise.IMain<object, pg.IClient>
     pgPool: Pool
+    transactionMode: object
+    
+    set pgp(value: pgPromise.IMain<object, pg.IClient>) {
+        requestLogger.info("SET PGP")
+        // Create a reusable transaction mode (serializable + read-only + deferrable):
```

Comment:
Is this example code, or properly dead code?

Reply:
Permanently dead, removed and buried it.

Reply:
Did you put a nice tombstone on it? :D